apps/guardian-ui/src/setup/SetupContext.tsx

```diff
@@ -237,9 +240,7 @@ export const SetupContextProvider: React.FC<SetupContextProviderProps> = ({
     useCallback(
       async ({ password: newPassword, myName, configs }) => {
         if (!password) {
-          if (!configGenParams) {
```

Comment:
This extra conditional was no longer necessary, now that we initialize `password` correctly.